cfme/base/__init__.py

```diff
@@ -126,21 +126,11 @@ def get_master(self):
         else:
             server = server_collection.find_by(is_master=True)[0]
 
-        # TODO This entire hack needs to be removed once we are able to edit the server name again
         try:
-            name = server.name
+            name = getattr(server, 'name')
         except AttributeError:
-            logger.error('The EVM has no name, setting it to EVM')
-            if (self.appliance.version == LATEST or
```

Comment:
@izapolsk I've tested locally to confirm that the upstream appliances have a name set